linux_sql/README.md

```diff
@@ -0,0 +1,217 @@
+#Linux Cluster Monitoring Agent
+## Introduction
+#### What does this project do?
+In this project, we are creating several shell scripts to gather and record the hardware information of the server, 
+as well as continuously monitor its resource usage on a minute-by-minute basis. The collected data will then be stored 
+in a PostgreSQL RDBMS for future reference and planning.
+
+#### Who are the users?
+The project was initially created for the Jarvis Linux Cluster Administration (LCA) team, which is responsible for 
+managing a Linux cluster of 10 servers. The LCA team needs to gather the hardware specifications of each server 
+and continuously monitor their resource usage in real-time. The collected data will be used to generate reports for 
+future resource planning and management.
+
+#### Technologies used?
+In this project, we utilize *git* for code management and utilize a *git repository* for code storage. The project was 
+constructed and tested on *Jarvis Remote Desktop (JRV)*. We create a *PSQL* instance using *docker* and connect to it through 
+the use of PSQL CLI tools. Additionally, we have written *bash scripts* and *DDL* SQL scripts to automate the process of 
+creating a docker instance, creating tables, and inserting data into the database.
+- **Git&Git Repository:** Manage and stored the code.
+- **Docker:** Provision the PSQL instance.
+- **JRV:** Remote desktop running centos 7.
+- **Bash Script&Linux Command:** Implement the project.
+- **PostgreSQL:** Store the server's hardware and real-time usage data.
```

Comment:
Since the "core" of the project is bash and psql, I would list them first. Git can be last because it is self-evident.